src/views/darkroom.c

```diff
@@ -2203,33 +2213,56 @@ void gui_init(dt_view_t *self)
     gtk_container_add(GTK_CONTAINER(dev->overexposed.floating_window), vbox);
 
     /** let's fill the encapsulating widgets */
+    /* preview mode */
+    GtkWidget *mode = dt_bauhaus_combobox_new(NULL);
+    dt_bauhaus_widget_set_label(mode, NULL, _("clipping preview mode"));
+    dt_bauhaus_combobox_add(mode, _("full gamut"));
+    dt_bauhaus_combobox_add(mode, _("any RGB channel"));
+    dt_bauhaus_combobox_add(mode, _("luminance only"));
+    dt_bauhaus_combobox_add(mode, _("saturation only"));
```

Comment:
As "any RGB channel" is what was done before, maybe we should we keep this as the default value instead of "full gamut" which is mostly equivalent to the other button (gammut check) ?

Reply:
Any RGB channel is really misleading as a default and is the reason why people do bad things now. It only shows oversaturation for whites, that is no actual over/under-exposure, but doesn't show oversaturation for valid luminances either. Full gamut has been chosen as a default because it is the combination of the 3 other modes, so it's a more complete and robust estimation of what's wrong in the picture.